src/ServerAdministration/WindowsService/WheelMUD.WindowsService.csproj

```diff
@@ -17,6 +17,9 @@
     </FileUpgradeFlags>
     <OldToolsVersion>3.5</OldToolsVersion>
     <UpgradeBackupLocation />
+    <IsWebBootstrapper>false</IsWebBootstrapper>
+    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
```

Comment:
Honestly don't know I have a lot of stuff installed and I use the ultimate version of visual studio on my home development box so maybe that's related.